<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RacaClass</class>
 <widget class="QMainWindow" name="RacaClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raca</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource/Raca.qrc">
    <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>输入关键词</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="leSearch">
        <property name="statusTip">
         <string>输入关键词按下回车以搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>结果</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvResult"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDeleteSelectedRow"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionExportDataBase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>搜索(&amp;S)</string>
    </property>
    <addaction name="actionUseRegex"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDeleteSelectedRow"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dwDetailInfo">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>详细信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTitle">
       <property name="statusTip">
        <string>选中行记录的标题</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>内容</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="teArticle">
       <property name="statusTip">
        <string>选中行记录的内容</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>标签</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwTag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="statusTip">
        <string>选中行记录的标签</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>创建日期</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leCreateTime">
       <property name="statusTip">
        <string>选中行记录的创建时间</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resource/Raca.qrc">
     <normaloff>:/ic_add.svg</normaloff>:/ic_add.svg</iconset>
   </property>
   <property name="text">
    <string>添加(&amp;A)...</string>
   </property>
   <property name="statusTip">
    <string>添加新文章</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于 Qt...</string>
   </property>
   <property name="statusTip">
    <string>关于 Qt</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUseRegex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>使用正则表达式(&amp;R)</string>
   </property>
   <property name="iconText">
    <string>使用正则表达式(R)</string>
   </property>
   <property name="toolTip">
    <string>使用正则表达式(R)</string>
   </property>
   <property name="statusTip">
    <string>使用正则表达式进行搜索</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource/Raca.qrc">
     <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
   </property>
   <property name="text">
    <string>关于 Raca...</string>
   </property>
   <property name="statusTip">
    <string>显示程序版本、版权信息等</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource/Raca.qrc">
     <normaloff>:/ic_settings.svg</normaloff>:/ic_settings.svg</iconset>
   </property>
   <property name="text">
    <string>选项(&amp;O)...</string>
   </property>
   <property name="statusTip">
    <string>更改工具选项</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDeleteSelectedRow">
   <property name="icon">
    <iconset resource="resource/Raca.qrc">
     <normaloff>:/ic_remove.svg</normaloff>:/ic_remove.svg</iconset>
   </property>
   <property name="text">
    <string>删除选中行(&amp;D)</string>
   </property>
   <property name="statusTip">
    <string>删除选中的行(可多选)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionExportDataBase">
   <property name="text">
    <string>导出数据库为CSV文件...</string>
   </property>
   <property name="statusTip">
    <string>将数据库的内容导出为多个CSV文件</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resource/Raca.qrc">
     <normaloff>:/ic_edit.svg</normaloff>:/ic_edit.svg</iconset>
   </property>
   <property name="text">
    <string>修改选中行(&amp;E)...</string>
   </property>
   <property name="statusTip">
    <string>修改选中行的内容</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource/Raca.qrc"/>
 </resources>
 <connections/>
</ui>
